cnxarchive/tests/data/data.sql

```diff
@@ -557,6 +572,7 @@ INSERT INTO module_files VALUES (4, 23, 'PhET_Icon.png');
 INSERT INTO module_files VALUES (6, 23, 'PhET_Icon.png');
 INSERT INTO module_files VALUES (9, 23, 'PhET_Icon.png');
 INSERT INTO module_files VALUES (10, 23, 'PhET_Icon.png');
+INSERT INTO module_files VALUES (20, 109, 'index.cnxml.html');
```

Comment:
A composite module is singularly owned by a collection. So it cannot be acquired outside the context of the collection.

If we really really had to provide a route to composite modules, I think we could. Given the singular nature of the content, we could assume that any request for a composite module could be queried against `collated_file_associations` in conjunction with `latest_modules` if the version is missing. This would find the unique entry for the content.